R/plotCCA.R

```diff
@@ -0,0 +1,504 @@
+#' Plot RDA or CCA object
+#'
+#'
+#' @param object a
+#'   \code{\link[SummarizedExperiment:SummarizedExperiment-class]{SummarizedExperiment}}, a
+#'   \code{\link[TreeSummarizedExperiment:TreeSummarizedExperiment-constructor]{TreeSummarizedExperiment}}
+#'   or a rda/cca object. The latter is the output of \code{\link[mia:runCCA]{calculateRDA}}.
+#' 
+#' @param dimred A string or integer scalar indicating the reduced dimension to
+#'   plot. This is the output of \code{\link[mia:runCCA]{runRDA}} and resides in
+#'   \code{reducedDim(tse, dimred)}.
+#'
+#' @param colour_by String specifying the name of a column in colData to colour by
+#'   (Default: NULL).
+#' 
+#' @param color_by Alias for `colour_by`.
```

Comment:
These are parameters of plotReducedDim I guess? Then I think it is better to remove them from here